earth_enterprise/src/portableserver/build.py

```diff
@@ -1,4 +1,4 @@
-#! /usr/bin/python
+#! /usr/bin/env python2.7
```

Comment:
This probably makes sense to be fully consistent between portable and everything else now....

Reply:
Yes, plus it was causing portable to be using python 2.6.